client/src/App.js

```diff
@@ -35,7 +36,8 @@ function App ({ location }) {
           <Route path="/login" component={LogIn} />
           <Route path="/googleSignUp" component={SignUpGoogle} email={email} />
           <Route exact path="/" component={SignUpEmail} handleEmailEntry={handleEmailEntry} />
-          <Route path="/scheduler/:user/:event_type" component={Scheduler} />  
+          <Route exact path="/scheduler/:user/:event_type" component={Scheduler} />  
```

Comment:
`event_type` seems to have been part of another branch but i also recommend turning this into `eventType` for consistency